core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -978,15 +1015,19 @@ class AuthorizerIntegrationTest extends AbstractAuthorizerIntegrationTest {
    * even if the topic doesn't exist, request APIs should not leak the topic name
    */
   @ParameterizedTest
-  @ValueSource(strings = Array("kraft"))
-  def testAuthorizationFetchV12WithTopicNotExisting(quorum: String): Unit = {
+  @CsvSource(value = Array("false", "true"))
```

Comment:
Could we adjust the comment above to sth like "Test that the fetch request fails with TOPIC_AUTHORIZATION_FAILED if the client doesn't have permission and topic name is used in the request. Even if the topic doesn't exist, we return TOPIC_AUTHORIZATION_FAILED to prevent leaking the topic name."? Ditto for `testAuthorizationForProduceRequestVersionLessThan13`.

Reply:
Updated it for both produce and fetch requests.